### PR TITLE
Fixed bug: Addressed issues with reply and edit comment features in the comment section.

### PR DESCRIPTION
**Pull request by kr-2003**
Addressed issues with reply and edit comment features in the comment section.

### Preview

https://github.com/OWASP/BLT/assets/96587705/edda8e0b-3b02-48f7-a89c-8a3a74dc30f8

### CODE REVIEW

**Comment by kr-2003** [comment]
@DonnieBLT Kindly review this PR.

**Review by DonnieBLT — CHANGES_REQUESTED** [review]
One small tweak then looks ok

File: .gitignore
[the remark this answers is not in the document]
Comment (kr-2003) [reply .gitignore:1]: Sure.

**Comment by kr-2003** [comment]
@DonnieBLT Please review it. I already did the required changes.

**Review by DonnieBLT — APPROVED** [review]

**Pull request merged**